apps/web/pages/settings/my-account/appearance.tsx

```diff
@@ -137,6 +151,56 @@ const AppearanceView = ({
   return (
     <div>
       <Meta title={t("appearance")} description={t("appearance_description")} borderInShellHeader={false} />
+      <div className="border-subtle mt-6 flex items-center rounded-t-lg border p-6 text-sm">
+        <div>
+          {/* TODO: support i18n: currently it's hard code now, we should support other languages in other PR */}
```

Comment:
you have to just add translation for english language in apps/web/public/static/locales/en/common.json. we'll add translations for other languages later.

Reply:
For example:- If you have to use text "Reason for reschedule".
add a key value pair in en/common.json "reason_for_reschedule" :  "Reason for reschedule". 

and use it here t("reason_for_reschedule") like you can see other parts of code

Reply:
Gotcha, I think I did that. I added a new key called "app_theme". However, the vscode extension kept saying something like "ERROR: "app_theme" is not found in source with language tag 'en'". And the website isn't reflecting the new wording as well. Still trying to figure out why.

<img width="671" alt="Screenshot 2023-12-02 at 12 43 01 PM" src="https://github.com/calcom/cal.com/assets/71842426/2c259bb9-bae1-46ec-9321-8850c790e2f5">

<img width="1032" alt="Screenshot 2023-12-02 at 12 44 30 PM" src="https://github.com/calcom/cal.com/assets/71842426/73ebd4f3-460d-439f-8461-011c1087d134">

Reply:
After I `yarn dx`. The website has reflected the changes. Nice. But the vscode extension warning is still there

Reply:
@swh00tw no issue. this happens sometimes. just make sure you have added all the keys to common.json that you are using even though you see this error in extension. it gets fixed after some time